lib/l10n/app_it.arb

```diff
@@ -0,0 +1,600 @@
+{
+  "appName": "Aves",
+  "welcomeMessage": "Benvenuto in Aves",
+  "welcomeOptional": "Opzionale",
+  "welcomeTermsToggle": "Accetto i termini e le condizioni",
+  "itemCount": "{count, plural, =1{1 elemento} other{{count} elementi}}",
+
+  "timeSeconds": "{seconds, plural, =1{1 secondo} other{{seconds} secondi}}",
+  "timeMinutes": "{minutes, plural, =1{1 minuto} other{{minutes} minuti}}",
+  "timeDays": "{days, plural, =1{1 giorno} other{{days} giorni}}",
+  "focalLength": "{length} mm",
+
+  "applyButtonLabel": "APPLICA",
+  "deleteButtonLabel": "CANCELLA",
+  "nextButtonLabel": "AVANTI",
+  "showButtonLabel": "MOSTRA",
+  "hideButtonLabel": "NASCONDERE",
+  "continueButtonLabel": "CONTINUA",
+
+  "cancelTooltip": "Annulla",
+  "changeTooltip": "Cambia",
+  "clearTooltip": "Cancella",
+  "previousTooltip": "Precedente",
+  "nextTooltip": "Successivo",
+  "showTooltip": "Mostra",
+  "hideTooltip": "Nascondi",
+  "actionRemove": "Rimuovi",
+  "resetButtonTooltip": "Reimposta",
+
+  "doubleBackExitMessage": "Tocca di nuovo «indietro» per uscire",
+  "doNotAskAgain": "Non chiedere di nuovo",
+
+  "sourceStateLoading": "Caricamento",
+  "sourceStateCataloguing": "Catalogazione",
+  "sourceStateLocatingCountries": "Localizzazione dei paesi",
+  "sourceStateLocatingPlaces": "Localizzazione dei luoghi",
+
+  "chipActionDelete": "Elimina",
+  "chipActionGoToAlbumPage": "Mostra negli album",
+  "chipActionGoToCountryPage": "Mostra nei Paesi",
+  "chipActionGoToTagPage": "Mostra nelle etichette",
+  "chipActionHide": "Nascondi",
+  "chipActionPin": "Fissa in alto",
+  "chipActionUnpin": "Rimuovi dall'alto",
+  "chipActionRename": "Rinomina",
+  "chipActionSetCover": "Imposta copertina",
+  "chipActionCreateAlbum": "Crea album",
+
+  "entryActionCopyToClipboard": "Copia negli appunti",
+  "entryActionDelete": "Elimina",
+  "entryActionConvert": "Convertire",
+  "entryActionExport": "Esportazione",
+  "entryActionRename": "Rinomina",
+  "entryActionRestore": "Ripristino",
+  "entryActionRotateCCW": "Ruota in senso antiorario",
+  "entryActionRotateCW": "Ruota in senso orario",
+  "entryActionFlip": "Capovolgere orizzontalmente",
+  "entryActionPrint": "Stampa",
+  "entryActionShare": "Condividi",
+  "entryActionViewSource": "Visualizza sorgente",
+  "entryActionViewMotionPhotoVideo": "Aprire foto in movimento",
+  "entryActionEdit": "Modifica",
+  "entryActionOpen": "Aprire con",
+  "entryActionSetAs": "Imposta come",
+  "entryActionOpenMap": "Mostra in app mappa",
+  "entryActionRotateScreen": "Ruota lo schermo",
+  "entryActionAddFavourite": "Aggiungi ai preferiti",
+  "entryActionRemoveFavourite": "Rimuovi dai preferiti",
+
+  "videoActionCaptureFrame": "Cattura fotogramma",
+  "videoActionMute": "Disattiva",
+  "videoActionUnmute": "Disattiva",
+  "videoActionPause": "Pausa",
+  "videoActionPlay": "Play",
```

Comment:
I saw you used `riproduzione` elsewhere. It doesn't work here? Are you sure you want to keep the English?